kymatio/scattering2d/scattering2d.py

```diff
@@ -107,26 +107,45 @@ def build(self):
         self.Psi = convert_filters(filters['psi'])
         self.Phi = convert_filters([filters['phi'][j] for j in range(self.J)])
 
-    def _type(self, _type):
+    def _apply(self, fn):
+        """
+            Mimicates the behavior of the function _apply() of a nn.Module()
```

Comment:
`s/Mimicates/Mimics` like everywhere. Maybe you can use `sed` lol

Reply:
https://stackoverflow.com/questions/11392478/how-to-replace-a-string-in-multiple-files-in-linux-command-line/20721292

Reply:
done, tks!!